unit-test/data/protections.json

```diff
@@ -11,12 +11,6 @@
         ],
         "scripts": []
     },
-    "temporary-storage": {
```

Comment:
I don't think we should be removing that?

Reply:
Agreed... the whole file should be removed in https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/728/ actually.